tests/test_extension.py

```diff
@@ -24,6 +24,7 @@ def test_get_config_schema():
     assert "scan_flush_threshold" in schema
     assert "scan_follow_symlinks" in schema
     assert "excluded_file_extensions" in schema
+    assert "included_file_extensions" in schema
```

Comment:
The included/excluded order is flipped here too.

Reply:
God, really sorry, I thought I had caught all of these. Fixed now.